.github/workflows/checkIssueInfo.yaml

```diff
@@ -0,0 +1,29 @@
+name: Check for missing information
+on: 
+  issues:
+    types: [opened]
+jobs: 
+  checkIssueInformation:
+    name: Check for missing information
+    runs-on: ubuntu-latest
+    env:
+      issue_incomplete: 'False'
+      issue_label: needs more info
+      issue_comment: This issue is missing information. Please, make sure to fill in all the necessary fields.
```

Comment:
This issue is missing information. Please, make sure to fill in all the necessary fields marked as `(Fill in here)` to help our team digest the issue.

Reply:
Sure, will update it